docs/lang/articles/contribution/dev_install.md

```diff
@@ -224,7 +224,7 @@ sudo systemctl restart docker
 
 - If `python3 setup.py develop/install` gives `permission denied` error, it means you're
   installing into system python without write permission. You can work around this by:
-  - `python3 setup.py --user develop/install`
+  - `python3 setup.py develop/install --user`
```

Comment:
I tried to use this command, yet got the following error message:
```python
>>> python3 setup.py develop/install
[]
['taichi', 'taichi.misc', 'taichi.tools', 'taichi.core', 'taichi.tests', 'taichi.snode', 'taichi.lang', 'taichi.aot']
invalid command name 'develop/install'
```

It seems that `develop/install` is not a registered command?